src/odbc/unittests/array.c

```diff
@@ -107,6 +116,10 @@ query_test(int prepare, SQLRETURN expected, const char *expected_status)
 			exit(1);
 		}
 
+		/* can't check status values if we hadn't asked for themm */
```

Comment:
typo: themm -> them

Reply:
Thanks for noticing, will (force) push to fix this soon (when the CI jobs finish, just to see if there are no unforeseen problems).